### PR TITLE
Add skeleton html for Support Page

### PR DESCRIPTION
**Pull request by ThomasVy**
Added the skeleton html page for the Support Page. No css and functionality is no there yet.

### CODE REVIEW

**Comment by bill-luu** [comment]
Looks like it works, there's some linting errors though

**Comment by ThomasVy** [comment]
Are you guys cool with the folder structure? 
And I also hate kebab-case

**Comment by bill-luu** [comment]
Thanks for pointing that out, I forgot to check that.

There's some inconsistency in terms of the folder names - `SupportUsPage` vs `support-us`. I think the telemetry server prefers the lower case naming, so not a bad idea

File: src/app/app.module.ts [hunk 1]
```diff
@@ -3,10 +3,21 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { SponsorComponent } from './components/support-us-page/sponsor/sponsor.component';
+import { DonateComponent } from './components/support-us-page/donate/donate.component';
+import { InKindDonationsComponent } from './components/support-us-page/in-kind-donations/in-kind-donations.component';
+import { SocialMediaComponent } from './components/common/social-media/social-media.component';
+import { SupportUsMainComponent } from './/components/support-us-page/support-us-main/support-us-main.component';
```
Comment (liconway) [inline src/app/app.module.ts]: Not very important but `.//components`

**Pull request merged**